Xamarin.Forms.ControlGallery.iOS/Info.plist

```diff
@@ -36,7 +36,7 @@
 		<string>Default-812h@3x.png</string>
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
```

Comment:
This change is necessary to programmatically change the style of the StatusBar (light or dark). It is not necessary to change the background color. How are we going to manage this case? I think it must be prominently in the documentation.

Reply:
it should be indicated in the documentation, because if this option is not enabled, then the color change simply does not work.